traveler-ios-ui/TravelerKitUI/ViewControllers/AvailabilityViewController.swift

```diff
@@ -42,6 +42,13 @@ open class AvailabilityViewController: UIViewController {
     private var datePickerCellIndexPath: IndexPath {
         return IndexPath(row: 1, section: 0)
     }
+    private var labelColor: UIColor {
```

Comment:
If only using this in cellForRowAt, just make it a function variable.  No need for it to be a class variable.